configs/14.0/packages/glibc/stage_1

```diff
@@ -140,6 +140,15 @@ atcfg_pre_configure()
 atcfg_configure()
 {
 	local base_target=$(find_build_target ${AT_BIT_SIZE})
+
+	#  In case of cross build, glibc's libcrypt will be  compiled
+	# because it´s required to build gcc.
+	if [[ ("${distro_fm}" == "redhat") ||
+		("${distro_fm}" == "suse" ) ||
+		("${DISTRO_ID}" == "ubuntu-20") ]]; then
+		disable_crypt=yes
+	fi
```

Comment:
The comment and the values being tested do not match.
Is the comment out-of-date?